src/shared/directives/conditional-show/conditional-show.directive.ts

```diff
@@ -31,18 +32,16 @@ export class ConditionalShowDirective implements AfterViewInit, OnDestroy {
   ngAfterViewInit() {
     if (this.caseField.show_condition) {
       this.condition = new ShowCondition(this.caseField.show_condition);
-      // console.log('FIELD: ' + this.caseField.id + ' init. Show condition: ' + this.caseField.show_condition);
       this.formGroup = this.formGroup || new FormGroup({});
       this.formField = this.formGroup.get(this.caseField.id);
-      // console.log('FIELD: ' + this.caseField.id + '. Is form field:' + this.formField + '. Event fields:', this.eventFields);
+
```

Comment:
Similar to above about Mario's wanting to leave it.

Reply:
well commented lines of code don't look great and leaving them so that the developers can uncomment and debug seems a bit lazy 😁

Reply:
I fully agree 😄 Please ask Mario why he wanted that in. I wanted to remove it too last time.